devel/kchmviewer/files/patch-use-webkit-and-dbus.diff

```diff
@@ -0,0 +1,71 @@
+diff --git a/src/src.pro b/src/src.pro
+index 2524b5e..f9e7098 100644
+--- a/src/src.pro
++++ b/src/src.pro
+@@ -40,7 +40,7 @@ SOURCES += config.cpp \
+     textencodings.cpp \
+     treeitem_toc.cpp \
+     treeitem_index.cpp
+-LIBS += -lchm -lzip
++#LIBS += -lchm -lzip
+ TARGET = ../bin/kchmviewer
+ CONFIG += threads \
+     warn_on \
+@@ -81,7 +81,7 @@ macx: {
+     HEADERS += kchmviewerapp.h
+     SOURCES += kchmviewerapp.cpp
+     QMAKE_INFO_PLIST=resources/Info.plist
+-    QMAKE_POST_LINK += cp resources/*.icns ${DESTDIR}/kchmviewer.app/Contents/Resources;
++    ICON = resources/kchmviewer.icns
+     LIBS += ../lib/libebook/libebook.a
+     POST_TARGETDEPS += ../lib/libebook/libebook.a
+ }
+@@ -102,7 +102,7 @@ win32-*: {
+     LIBS += -lwsock32 -loleaut32
+ }
+ 
+-unix:!macx: {
++unix: {
+ 
+     QT += dbus
+     HEADERS += dbus_interface.h
+@@ -112,6 +111,9 @@ unix: {
+     POST_TARGETDEPS += ../lib/libebook/libebook.a
+ }
+ 
++LIBS += -L@PREFIX@/lib -lchm -L@PREFIX@/lib/ -lzip
```

Comment:
Why a double `@PREFIX@`? Doesn't one suffice? (Just curious: why do you first comment the addition of libraries out and add them back in later?)